cookin/app/views/index_view.py

```diff
@@ -1,6 +1,58 @@
 from django.shortcuts import render
 from app.forms import SearchForm
 from app.models import Recipe
+from app.models import Rating
+from django.utils.text import slugify
+
+def average_cost(recipe):
```

Comment:
This function seems to belong in the recipe model. See the total_time function in the recipe model.